spec/src/main/asciidoc/core/beanmanager_lite.asciidoc

```diff
@@ -0,0 +1,268 @@
+[[programmatic_access]]
+
+== Programmatic access to container
+
+CDI 3.0 used to provide a single `BeanManager` object, which allows access to many useful operations.
```

Comment:
This is not only true for CDI 3.0 though. I would say: The CDI version prior to 4.0

Reply:
If you could suggest a way how to phrase this without the historical reference, that would be best. I admit I didn't give it much thought, I'll think about it again.

Reply:
`Prior to CDI 4.0, there used to be a single ....`?